iocore/net/UnixNetVConnection.cc

```diff
@@ -1273,14 +1273,21 @@ UnixNetVConnection::connectUp(EThread *t, int fd)
     con.is_bound     = true;
   }
 
+  if (check_emergency_throttle(con)) {
```

Comment:
Since you are touching this code, could you please change the return type of ``check_emergency_throttle`` to ``bool``?